client/js/store/modules/auth.js

```diff
@@ -107,10 +109,16 @@ export default {
         );
       });
     },
-    async requestStatus(context) {
-      const { location } = window;
-      const isLobby = location.hash.startsWith('#/lobby');
-      const isGame = location.hash.startsWith('#/game');
+    async requestStatus(context, { router, route }) {
+      const { state } = context;
+
+      if (!router || !route || state.authenticated !== null) {
+        return;
+      }
```

Comment:
Need a new check here to make sure router & route exist, and that the user's auth status isn't anything but `null`.

Reply:
How might we call this before router has loaded and what happens in that case when we return early? Does the page load later and detect the null auth?